icp:
  max_correspondence_distance: 3.0
  maximum_iterations: 50
  transformation_epsilon: 0.00000001
  euclidean_fitness_epsilon: 0
